pkgs/tools/misc/timg/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, pkgs, fetchFromGitHub }:
```

Comment:
although this is possible, its preferred to explicitly import the packages and not the whole pkgs set.

Reply:
Please do not use pkgs as an input.